src/Config/ConfigFile.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Codelicia\Xulieta\Config;
+
+use Symfony\Component\Config\Definition\Processor;
+use Symfony\Component\Yaml\Yaml;
+use function array_filter;
+use function current;
+use function file_get_contents;
+use function realpath;
+
+final class ConfigFile
+{
+    /**
+     * @return string[]
+     */
+    public static function loadInDirectory(string $dir) : array
+    {
+        $configFiles = current(array_filter(
```

Comment:
Wouldn't `current` default to the last entry of this array? Or would it pick the first? What does mutation testing tell us here?